testnets/nomictestnet/assetlist.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nomictestnet",
+  "assets": [
+    {
+      "description": "Nomic's native token.",
+      "denom_units": [
+        {
+          "denom": "unom",
+          "exponent": 0
+        },
+        {
+          "denom": "nom",
+          "exponent": 6
+        }
+      ],
+      "base": "unom",
+      "name": "Nomic",
+      "display": "nom",
+      "symbol": "NOM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nomic/images/nom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nomic/images/nom.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nomic/images/nom.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nomic/images/nom.svg",
+          "theme": {
+            "primary_color_hex": "#6404fc"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Bitcoin. On Cosmos.",
+      "denom_units": [
+        {
+          "denom": "usat",
+          "exponent": 0
+        },
+        {
+          "denom": "nbtc",
+          "exponent": 14
+        }
+      ],
+      "base": "usat",
+      "name": "Nomic Bitcoin",
+      "display": "nbtc",
+      "symbol": "nBTC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nomic/images/nbtc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nomic/images/nbtc.svg"
+      },
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "bitcoin",
+            "base_denom": "sat"
+          },
+          "provider": "Nomic"
+        }
+      ],
+      "images": [
+        {
```

Comment:
Please use image_sync, pointing to the mainnet asset. Would be good to also define the trace back to mainnet nom, using type: "test-mintage"